README.md

```diff
@@ -164,11 +164,22 @@ Have a look at [the milestones](https://github.com/kubernetes-incubator/external
 
 ## Contributing
 
-We encourage you to get involved with ExternalDNS, as users as well as contributors. Read the [contributing guidelines](CONTRIBUTING.md) and have a look at [the contributing docs](docs/contributing/getting-started.md) to learn about building the project, the project structure, and the purpose of each package.
+We encourage you to get involved with ExternalDNS, as users, contributors or as new maintainers that can take over some parts like different providers and help with code reviews.
 
-For an overview on how to write new Sources and Providers check out [Sources and Providers](docs/contributing/sources-and-providers.md).
+Providers which currently need maintainers:
+
+* Azure
+* Cloudflare
+* Digital Ocean
+* Google Cloud Platform
+
+Any provider should having at least one maintainer for each new provider would be really great.
```

Comment:
Any provider should have at least one maintainer. It would be nice if you run it in production, but it is not required. 
You should check changes and make sure your provider is working correctly.

Reply:
It would be also great to have an automated end-to-end test for different cloud providers, so help from kubernetes maintainers and their idea on how this can be done would be valuable